nixos/modules/services/misc/gitolite.nix

```diff
@@ -81,6 +104,12 @@ in
       script = ''
         cd ${cfg.dataDir}
         mkdir -p .gitolite/logs
+        >.gitolite.rc gitolite print-default-rc
```

Comment:
I didn't know IO redirection could be done at the start of the line.

Reply:
It seems to work in the middle of the line as well.

Reply:
Do you mind changing the IO redirect to the end of the line? My guess is that having it at end-of-line is more common and people reading the code won't have to stop and think about what the code does if it looks more familiar. (At least that's how I feel.)

Reply:
Well, placing the redirects at the beginning creates prominent similarity between the commands that write to the file, therefore visually "grouping" them. This was my motivation for placing redirects like this. However, a predictable, well-known way of doing things is probably more valuable.